plugins/scaffolder/src/components/TemplateList/TemplateList.tsx

```diff
@@ -25,7 +25,11 @@ import { useEntityListProvider } from '@backstage/plugin-catalog-react';
 import { Link, Typography } from '@material-ui/core';
 import { TemplateCard } from '../TemplateCard';
 
-export const TemplateList = () => {
+export type TemplateListProps = {
+  templateCard?: Function | undefined;
```

Comment:
Let's use a proper component type for this, so some form of `(props: Props) => JSX.Element | null`. With that it's best to call the prop `TemplateCardComponent` as well

Reply:
Sure, I address this comment on [9388571](https://github.com/backstage/backstage/pull/6829/commits/9388571643faa5d074b49133dce413dca91137ea)